calculator/src/App.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+// import logo from './logo.svg';
+// import './App.css';
+import Display from './components/Display';
+
+class App extends React.Component {
+  state = { 
+    displayValue: 0, 
+    prevValue: "", 
+    operation: "",
+    clearEntry: false,
+    calculatedSolution: false
+  };
+
+  handleNumberClick = (e) => {
+    if (this.state.displayValue) {
+      this.setState({displayValue: this.state.displayValue + e.target.value})
+    } else {
+      this.setState({displayValue: e.target.value})
+    }
+  }
+
+  handleOperations = (e) => {
+    this.setState({
+      operation: e.target.value, 
+      prevValue: this.state.displayValue, 
+      displayValue: ""
```

Comment:
you don't need to clear your display value when you press the operation button. You should just keep the original display value until another button has been pressed after the operator. Otherwise the calculator looks a little weird.